src/Illuminate/Foundation/helpers.php

```diff
@@ -470,6 +470,27 @@ function factory()
     }
 }
 
+if (! function_exists('has_errors')) {
+    /**
+     * Returns true if session has errors, and optionally key-specific errors.
+     *
+     * @param  string|null  $key
+     * @param  string|null  $bag
+     * @return bool
+     */
+    function has_errors($key = null, $bag = 'default')
```

Comment:
Should be changed back to `$bag = null`, then add something like `$bag = $bag ?: 'default'` at beginning of code. One should be able to programmatically use the default bag by passing `null`.